### PR TITLE
Improve ova build - packerless building

### LINKED ISSUE #669: Change default appliance password
Change to something that's easier to enter on DCUI
https://github.com/vmware/vic-product/blob/master/installer/packer/packer-vic.json#L8

### LINKED ISSUE #708: Improve OVA build
**User Statement:**

As a developer I want to be able to build without using Packer

**Details:**
create a base photon install: https://github.com/vmware/cove/blob/master/build/baseimage/stage.sh
customizations happen here: https://github.com/vmware/cove/blob/master/build/baseimage/install.sh
there is still 1 step missing that is converting the raw image into a vmdk
https://hub.docker.com/r/casualjim/qemu-img/

**Acceptance Criteria:**
- [ ] Local build of OVA doesn't use Packer
- [ ] Documentation

### PR DESCRIPTION
**Pull request by morris-jason**
Fixes #708, Fixes #669 

@casualjim, @frapposelli, @andrewtchin, @anchal-agrawal can you please review?

Extends @casualjim's build scripts for a base photon vmdk into
a group of build scripts for provisioning an ova. This removes
the need for packer.

Build process hasn't changed - use ./build/build.sh from the installer directory.
The new make target is `ova`, `ova-build`, and `ova-release`.

### CODE REVIEW

File: installer/build/scripts/systemd/dcui_override.conf [hunk 1]
```diff
@@ -2,7 +2,7 @@
 Description=
 Description=DCUI
 After=
-After=systemd-networkd.service
+After=network-online.target
```
Comment (andrewtchin) [inline installer/build/scripts/systemd/dcui_override.conf]: 👍

File: installer/build/scripts/provision_harbor.sh [hunk 2]
```diff
@@ -14,63 +14,17 @@
...
-dch_image="vmware/dch-photon:1.13"
-dch_tag="dch-photon:1.13"
-docker pull $dch_image
-docker run -d --name dch-push -v /data/harbor/registry:/var/lib/registry -p 5000:5000 vmware/registry:2.6.2-photon
-docker tag $dch_image 127.0.0.1:5000/default-project/$dch_tag
-sleep 3
-docker push 127.0.0.1:5000/default-project/$dch_tag
-docker rm -f dch-push
+echo "Provisioning Harbor ${BUILD_HARBOR_FILE}"
+cat /etc/cache/${BUILD_HARBOR_FILE}  | tar xz -C /var/tmp
```
Comment (andrewtchin) [inline installer/build/scripts/provision_harbor.sh]: 👍

File: installer/build/scripts/system_settings.sh [hunk 3]
```diff
@@ -14,39 +14,20 @@
...
-# Disable password expiration for root
-chage -I -1 -m 0 -M 99999 -E -1 root
-
 # Enable systemd services
-systemctl daemon-reload
 systemctl enable toolbox.service
 systemctl enable docker.service
 systemctl enable data.mount repartition.service resizefs.service getty@tty2.service
-systemctl enable chrootpwd.service sshd_permitrootlogin.service vic-appliance.target
+systemctl enable sshd_permitrootlogin.service firstboot.service vic-appliance.target
```
Comment (andrewtchin) [inline installer/build/scripts/system_settings.sh]: What takes the place of chrootpwd?

File: installer/build/scripts/upgrade/upgrade.sh [hunk 4]
```diff
@@ -297,7 +299,7 @@ function performAdmiralUpgrade {
     --log-driver=json-file \
     --log-opt max-size=1g \
     --log-opt max-file=10 \
-    "vmware/admiral:vic_v1.1.1"
+    "vmware/admiral:vic_${BUILD_ADMIRAL_RELEASE}"
```
Comment (andrewtchin) [inline installer/build/scripts/upgrade/upgrade.sh]: This should always be 1.1.1 at least for upgrading from 1.1.x. We will have to think about how to do upgrade for following versions

**Comment by andrewtchin** [comment]
```	build/baseimage/cache/
	build/baseimage/root/installer.env
```
I think these need to be added to .gitignore and make clean

File: installer/build/scripts/upgrade/upgrade.sh (hunk 4 above)
Comment (morris-jason) [reply installer/build/scripts/upgrade/upgrade.sh]: Ok. That var is set in build.sh, which is 1.1.1.

**Pull request merged**